turbolift/methods/archive.py

```diff
@@ -35,8 +35,14 @@ def start(self):
             self._put_container()
 
         LOG.info('Uploading Archive...')
-        with indicator.Spinner(**self.indicator_options):
-            self._upload(**archive)
+        upload_objects = self._return_deque()
+        archive_item = self._encapsulate_object(
+            full_path=archive['local_object'],
+            split_path=os.path.dirname(archive['local_object'])
+        )
+        upload_objects.append(archive_item)
```

Comment:
This looks good to me. Testing by hand seems to work. I've not had a lot of time to improve on tbl3 but I hope that changes soon.